Assets/Plugins/Source/EOSManager.cs

```diff
@@ -527,7 +527,7 @@ public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
 
                 string configDataAsString = "";
 
-#if UNITY_ANDROID
+#if UNITY_ANDROID && !UNITY_EDITOR
```

Comment:
Thanks for the discovery.
Though this might break for android on editor_windows.
!UNITY_EDITOR_OSX might fix the problem

Reply:
I don't think you wan't to run android code in the editor on windows either?
That is only ever useful in standalone builds (on device) as far as I know...

Reply:
With platform set to Android, it was running on editor, partially using android utilities, that is why we wouldn't want to change UNITY_EDITOR_WIN part.
Though it seems that with !UNITY_EDITOR android on windows editor still works, and after reading what the code it makes sense to use !UNITY_EDITOR.
Thanks for the PR.